docs/src/tutorials/supported_operations.md

```diff
@@ -20,6 +20,12 @@ The specification for the operation files is described in `thin-edge.io` specifi
 
 Supported operations are declared in the cloud specific subdirectory of `/etc/tedge/operations` directory.
 
+### `thin-edge.io child` Supported Operations
+
+When a new child device is added, the boot-straping of child device will create `/etc/tedge/operations/c8y/<child-device>` directory to store the supported operations by that particular child device.
+It creates a file per Supported Operation. The operation can be dynamically added and removed. A special file placed in `/etc/tedge/operations/c8y/<child-device>` directory
+will indicate that an operation is supported by that `child device`.
+
```

Comment:
I would remove this section or move it under "Custom Supported Operations".

I'm saying  that because this section is giving details of what is under `c8y` while cloud specific operations are introduced after.

Reply:
Moved below. Also, made it more generic by changing `/etc/tedge/operations/c8y/<child-device>` to `/etc/tedge/operations/<cloud-provider>/<child-device>`.